Changes between KDE 3.5.10 tag and KDE 3.5 branch r975372.
Only looking at subdirectory kdebase/kcontrol here.
Logs since r849627 of the branch but this might be wrong.

r852629 | lunakl
Update wallpapers correctly when adding new screen with xrandr1.2 (bnc#329221).
  M background/bgrender.cpp
r863251 | ianjo
Backport of r786315 to 3.5 branch: Fix rarely hit path where glXDestroyContext would be called twice.
  M info/opengl.cpp
r886803 | tyrerj
BUG:84101
  M arts/arts.cpp
r975372 | dfaure
apply gcc-4.4 compilation fix from http://cvs.pld-linux.org/cgi-bin/cvsweb.cgi/packages/kdebase/kdebase-gcc44.patch. ...
  M kfontinst/kfontinst/Fontmap.cpp


Index: background/bgrender.cpp
===================================================================
--- kdebase/kcontrol/background/bgrender.cpp	 (.../tags/KDE/3.5.10)	(revision 850549)
+++ kdebase/kcontrol/background/bgrender.cpp	 (.../branches/KDE/3.5)	(revision 975372)
@@ -1163,9 +1163,8 @@
         m_pPixmap = new QPixmap(m_size);
         m_pPixmap->fill(Qt::black);
     }
-    
-    for (unsigned i=0; i<m_numRenderers; ++i)
-        m_renderer[i]->desktopResized();
+
+    initRenderers();    
 }
 
 
Index: info/opengl.cpp
===================================================================
--- kdebase/kcontrol/info/opengl.cpp	 (.../tags/KDE/3.5.10)	(revision 850549)
+++ kdebase/kcontrol/info/opengl.cpp	 (.../branches/KDE/3.5)	(revision 975372)
@@ -608,7 +608,6 @@
    }
    else {
       kdDebug() << "Error: glXMakeCurrent failed\n";
-      glXDestroyContext(dpy, ctx);
    }
 
    glXDestroyContext(dpy, ctx);
Index: arts/arts.cpp
===================================================================
--- kdebase/kcontrol/arts/arts.cpp	 (.../tags/KDE/3.5.10)	(revision 850549)
+++ kdebase/kcontrol/arts/arts.cpp	 (.../branches/KDE/3.5)	(revision 975372)
@@ -214,7 +214,7 @@
 	config = new KConfig("kcmartsrc");
 	load();
 
-	suspendTime->setRange( 0, 999, 1, true );
+	suspendTime->setRange( 1, 999, 1, true );
 
 	connect(startServer,SIGNAL(clicked()),this,SLOT(slotChanged()));
 	connect(networkTransparent,SIGNAL(clicked()),this,SLOT(slotChanged()));
Index: kfontinst/kfontinst/Fontmap.cpp
===================================================================
--- kdebase/kcontrol/kfontinst/kfontinst/Fontmap.cpp	 (.../tags/KDE/3.5.10)	(revision 850549)
+++ kdebase/kcontrol/kfontinst/kfontinst/Fontmap.cpp	 (.../branches/KDE/3.5)	(revision 975372)
@@ -44,7 +44,7 @@
 
 using namespace std;
 
-static char * findSpace(char *str)
+static const char * findSpace(const char *str)
 {
     while(str && *str!=' ' && *str!='\t')
         str++;
@@ -65,7 +65,7 @@
     char a[constMaxLen+1],
          b[constFileMaxLen+1];
 
-    char *slash1=strchr(line, '/'),
+    const char *slash1=strchr(line, '/'),
          *space1=slash1 ? findSpace(slash1) : NULL, //strchr(slash1, ' ') : NULL,
          *ob=slash1 ? strchr(slash1, '(') : NULL,
          *cb=ob ? strchr(ob, ')') : NULL,
